docs/getting_started.md

```diff
@@ -70,26 +70,31 @@ To run the scrapers for today:
 yarn start
 ```
 
-### Run only one scraper
+### Run scrapers for only one location
```

Comment:
So this is not "only one location" anymore, but this is the last minor thing I could find.

Reply:
Yeah I dithered on that one ... is the US one location?  Actually, I'll call it "Run selected scrapers."  Thanks!

Reply:
Updated, thank you!